rust/src/delta.rs

```diff
@@ -658,13 +665,15 @@ impl DeltaTable {
         storage_backend: Box<dyn StorageBackend>,
     ) -> Result<Self, DeltaTableError> {
         let log_path_normalized = storage_backend.join_path(table_path, "_delta_log");
+        let commit_path_normalized = storage_backend.join_path(table_path, "_prepared_commits");
         Ok(Self {
```

Comment:
I;m still not confident whether it is a right way to do it.
E.g. before, we stored tmp files for fs within a root dir, but now, since we plan to introduce a recovery option, it'd be handy to store these files within its separate directory. However, since rename for fs is atomic, we might not need that. So does it make sense to move this into storage level code?

Reply:
yeap, let's not bring new concept, e.g. new dir, and place within table_path, later we'll se how that'll work for us

Reply:
actually, on the second thought I've placed them under log_path, because a) it so connected to log rather then files, b) iirc data file names are arbitary hence we won't have a name collision if someone would try to name a file `_commit_...`